### ISSUE TITLE
storing a style model

### ISSUE THREAD

**Issue by djdjoko** [jcjohnson/neural-style#79]
Is it possible to learn a style, save it and reapply it later to other content images in order to decrease processing time?

**Comment by 3DTOPO** [comment]
I would love to hear what others have to say about it, but I suspect not. If I understand the paper correctly, it convolves from seed noise (and/or the image) to a style image. That would mean every convolution is unique to input image / style (and settings of course).

**Comment by jcjohnson** [comment]
@3DTOPO You are correct - there is not a good way to save the style with the current algorithm.To generate an image, we run an optimization problem whose variables are the values of the pixels in the generated image. We use a convolutional neural network to do this, but the weights of the network are fixed; they are just used to evaluate the generated image.
